tests/quotes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -57,11 +57,11 @@ exports[`test strings.js 1`] = `
 \"\'\";
 
 \'\"\';
-\'\"\';
+\'\\\"\';
```

Comment:
Before, 

```js
'\"' -> '"'
```

now, we don't touch strings if we don't change their quotes so it stays the same with an extra (useless) `\`

```js
'\"' -> '"'
```